testcases/pkcs11/sess_perf.c

```diff
@@ -183,7 +183,7 @@ int do_SessionPerformance(unsigned int count)
         rc = create_des_encrypt_context(&(t[i].hsess), &(t[i].hkey));
         if (rc == FALSE) {
             show_error("create_aes_encrypt_context", (CK_RV) 0);
-            return FALSE;
+            goto ret;
```

Comment:
problem is function returned boolean before and now rcs of callees. As noted in my other comment i think it would make sense to have a seperate return code var for  the function return, initialized to false.

Reply:
Well, in all cases there is a check for 

> if (rc == FALSE)

 just before the goto, so rc is FALSE in all those cases.

Reply:
somehow missed that